ports/sentry-native/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "sentry-native",
-  "version-semver": "0.4.15",
+  "version-semver": "0.4.16",
```

```suggestion
  "version": "0.4.16",
```
This library isn't a semver library; it should be switched to `"version"`.

---

Fixed